File: C/exercicio7.c
#include <stdio.h>

int main(void)
{
  float lado, area, dobro;
  do
    {
      printf("Digite o lado de um quadrado: ");
        scanf("%f", &lado);
      if (lado < 1)
      {
        printf("Digite um valor válido!\n");
      }
    }
    while (lado < 1);
  area = lado * lado;
  dobro = area * 2;
  printf("A área do quadrado e seu dobro são, aproximadamente: %.2f e %.2f\n", area, dobro);
  return 0;
}

File: Python/avulsos/exercicio7.py
import math

lado = float(input("Digite o valor do lado de um quadrado: "))
while lado < 1:
  print("Digite um valor válido!")
  lado = float(input("Digite o valor do lado de um quadrado: "))
area = math.pow(lado, 2)
dobro = area * 2
print(f"A área do quadrado de lado {lado} e seu dobro, respectivamente: {area:.2f}, {dobro:.2f}")